include/singa/core/tensor.h

```diff
@@ -440,7 +440,14 @@ void Mult(const Tensor &A, const Tensor &B, Tensor *C);
 template <typename SType>
 void Mult(const SType alpha, const Tensor &A, const Tensor &B, const SType beta,
           Tensor *C);
-
+/// Do high dimension matrix multipication on the tensor shape. result = A * B
```

Comment:
what's the difference of HighDimMult and GEMM or GEMV?

Reply:
The GEMM and GEMV(which use cblas_sgemm or cblas_sgemm) could only calculate the 2D matrix or vector multiplication. When the matrix becomes 3D or above, here I use the HighDimMult to do the multiplication.